requirements.txt

```diff
@@ -1,3 +1,4 @@
 flask==2.0.3
 requests
 logicmonitor_sdk==2.0.163
+werkzeug==2.2.2
```

Comment:
Yeah, you can add Python 3.8 in the workflow like this:
https://github.com/StackStorm-Exchange/stackstorm-openai/blob/6e492e6b945fe8b77c8a3516d965a2904383f90e/.github/workflows/build_and_test.yaml#L12-L17
There is also a release workflow in the same dir that needs to be updated too.

Though with dropping python 3.6, this pack probably needs a major release bump like `2.0.0`.

But before that, from the https://werkzeug.palletsprojects.com/en/2.2.x/changes/#version-2-0-3, `2.0.3` is the latest version that supported python 3.6.
Can we try that first?

Reply:
cc @lm-ydubler

Reply:
werkzeug v2.0.3 does work. I have also tested this on my local docker env and production environment and the pack is working as expected. I am happy for the 2.0.3 to be merged in for now and we can look at a major release update later on. That way LogicMonitor Developers can have their say as well

Reply:
Nice! Left a few final minor comments